modules/websearch/lib/websearch_templates.py

```diff
@@ -167,6 +168,9 @@ class Template:
         'wl': (int, CFG_WEBSEARCH_WILDCARD_LIMIT),
         }
 
+    if CFG_WEBSEARCH_OBELIX_REDIS:
+        search_results_default_urlargd['rm'] = (str, "rrm")
```

Comment:
Okay, it sounds like a solid solution (as long as you put it as default on the primary collection). 

Should I modify the code then?